ui/app/templates/components/transit-key-actions.hbs

```diff
@@ -1,17 +1,16 @@
 {{#if (eq selectedAction 'rotate')}}
   {{#if key.canRotate}}
-    {{#confirm-action
-        buttonClasses="toolbar-link"
-        onConfirmAction=(action "doSubmit")
-        confirmMessage=(concat 'Are you sure you want to rotate "' key.id '"?')
-        confirmButtonText="Confirm rotation"
-        cancelButtonText="Cancel"
-        messageClasses="is-block-mobile has-text-grey"
-        data-test-transit-key-rotate=true
-      }}
+    <ConfirmAction
+      @buttonClasses="toolbar-link"
+      @confirmTitle="Rotate this key?"
+      @confirmMessage="After rotation, new plaintext requests will be encrypted with the new version of the key."
```

Comment:
I don't think we want to mention encryption here because the key may not support encrypt / decrypt. Maybe "After rotation, all key actions will default to using the newest version of the key." ?

Reply:
I think this is the only one I copied word for word from the docs (https://www.vaultproject.io/api/secret/transit/index.html#rotate-key), but I do like the change better. I wonder if we should add that detail there then